aws/rust-runtime/aws-sigv4/src/http_request/uri_path_normalization.rs

```diff
@@ -232,6 +233,15 @@ mod tests {
         );
     }
 
+    // The CRT does this so I figured we should too.
+    #[test]
+    fn normalize_uri_path_should_merge_mulitple_subsequent_slashes_into_one() {
```

Comment:
```suggestion
    fn normalize_uri_path_should_merge_multiple_subsequent_slashes_into_one() {
```

Reply:
wow, good catch! I had to read it a few times to see the difference.